local-antora-playbook.yml

```
@@ -17,7 +17,7 @@ content:
   - url: https://github.com/redpanda-data/docs
     branches: [v/*, api, shared, site-search,'!v-end-of-life/*']
   - url: https://github.com/redpanda-data/cloud-docs
-    branches: 'main'
+    branches: '25.1-rpk'
```

```suggestion
    branches: 'main'
```
to commit before merge